src/ayb_db/models.rs

```diff
@@ -173,6 +173,37 @@ pub struct Entity {
     pub links: Option<Vec<Link>>,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct PartialEntity {
+    pub slug: Option<String>,
```

Comment:
are things like slug/entity_type options, or only the display_name onward?

Reply:
Only the `display_name`, I forgot to delete that from something I tested :p